3party/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if (NOT WITH_SYSTEM_PROVIDED_3PARTY)
   set(EXPAT_DTD OFF)
   set(EXPAT_NS ON)
   add_subdirectory(expat/expat)
+  add_library(expat::expat ALIAS expat)
```

Comment:
This change would also be great to test and merge separately.

Reply:
Ditto wouldn't want to dive into rebase-cherry-pick-hell